tests/integration/WooCommerce/SubscriptionTest.php

```diff
@@ -49,8 +50,9 @@ public function _before() {
         },
       ]
     );
+    $subscribersRepository = $this->diContainer->get(SubscribersRepository::class);
```

Comment:
In this file there are a couple usages of `Subscriber::getCurrentWPUser()`, is it possible to refactor them? Then this method will be unused and we can deprecate it (in case it's used by 3rd parties) and later remove.

Reply:
I don't know how I could forget. Thanks